src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -903,34 +895,34 @@ Dictionary AdaptiveTileRendererFactory::get_params_metadata()
         "batch_size",
         Dictionary()
             .insert("type", "int")
-            .insert("default", "8")
+            .insert("default", "16")
             .insert("label", "Batch Size")
             .insert("help", "How many samples to render before each noise calculation"));
 
+    metadata.dictionaries().insert(
+        "min_samples",
+        Dictionary()
+            .insert("type", "int")
+            .insert("default", "0")
+            .insert("label", "Min Samples")
+            .insert("help", "Amount of uniform samples to render before adaptive samples"));
```

Comment:
Replace "Amount of" by "Number of".
Replace "before adaptive samples" by "before adaptive sampling".